docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2295.md

```diff
@@ -0,0 +1,34 @@
+# "Custom" capabilities with configured job parameters
+
+Test verifies that 
+- administrator shall be able to add params field to the capability - list of key/values. 
+- parameters are set as the parameters for a job.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- Test tool is available to the non-admin user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Open the **PREFERENCES** tab | |
+| 4 | Click the **Launch** tab | |
+| 5 | Find the **`launch.capabilities`** preference | |
+| 6 | Save the preference value, then clear and input the following json <ul> `{` <ul> `"testCapability1": {` <ul> `"description": "Custom test capability 1"`, <br> `"commands": [` <ul> `"echo testLine1"`, <br> `"echo 'testLine1' > ~/testFile1.txt"` </ul> `],` <br> `"params": {` <ul> `"MY_PARAM1": "MY_VALUE1"`, <br> `"MY_BOOLEAN_PARAM": true,` <br> `"MY_NUMBER_PARAM": 1` </ul> `}` </ul> `},` <br> `"testCapability2": {` <ul> `"description": "Custom test capability 2",` <br> `"commands": [` <ul> `"echo testLine2",` <br> `"echo 'testLine2' > ~/testFile1.txt"` </ul> `],` <br> `"params": {` <ul> `"MY_PARAM2": "MY_VALUE2"`, <br> `"MY_BOOLEAN_PARAM": false,` <br> `"MY_NUMBER_PARAM": 2` </ul> `}` </ul> `},` </ul> `}` </ul> `}` |
```

Comment:
> `"MY_NUMBER_PARAM": 2` </ul> `}` </ul> `},` </ul> `}` </ul> `}`

Fix to: `"MY_NUMBER_PARAM": 2` </ul> `}` </ul> `}` </ul> `}`

Reply:
Done